src/pyird/utils/aperture.py

```diff
@@ -29,11 +29,15 @@ def __init__(self, trace_function, y0, xmin, xmax, coeff):
 
     def mask(self):
         """mask image
-        
+
         Returns:
             mask image
 
         """
         return trace(self.trace_function, self.y0, self.xmin, self.xmax, self.coeff)
 
-    
+    def mmf2(self):
```

Comment:
Can you add an explanation for `mmf2`?

Reply:
After revising this, I will merge this PR to develop.

Reply:
I added an explanation.